website/components/remote-plugin-docs/server.js

```diff
@@ -30,12 +27,15 @@ async function generateStaticProps(
   const navNode = getNodeFromPath(pathToMatch, navData, localContentDir)
   const { filePath, remoteFile, pluginTier } = navNode
   //  Fetch the MDX file content
-  const [err, mdxString] = filePath
-    ? //  Read local content from the filesystem
-      [null, fs.readFileSync(path.join(process.cwd(), filePath), 'utf8')]
-    : // Fetch remote content using GitHub's API
-      await fetchGithubFile(remoteFile)
-  if (err) throw new Error(err)
+  const mdxString = remoteFile
+    ? remoteFile.fileString
+    : fs.readFileSync(path.join(process.cwd(), filePath), 'utf8')
+  // Construct the githubFileUrl, used for "Edit this page" link
+  // Note: for custom ".docs-artifacts" directories, the "Edit this page"
+  // link will lead to the artifact file rather than the "docs" source file
```

Comment:
How does this change after the zip approach, since we no longer have artifact files?

Reply:
With the ZIP approach as proposed in the RFC, we have the requirements that 1) the `docs` folder lives at the root of the plugin repo and 2) the `docs.zip` is just a `.zip` of said folder. So we can easily infer the original source file that we link to.

That being said I definitely slipped up here, this comment feels a bit out-of-date. I think it might also make sense to move the URL construction back out here for consistency - I'll adjust this 👍

Reply:
✅ Adjusted the comments here, to reflect that we grab the `sourceUrl` off the `remoteFile` object.